src/js/trove/require-node-dependencies.js

```diff
@@ -0,0 +1,34 @@
+//
+// When building a web-standalone, browserify will parse this file
+// and produce a version which include each dependency that is required()
+//
```

Comment:
Having to maintain this file will be annoying, but I can't think of another way without using browserify in the compiler (see my earlier comment on this PR for why that's problematic)